Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -96,8 +104,23 @@ bool CheckClassExistsWithCorrectionForBlueprints(FString& ClassName)
 	return false;
 }
 
-bool GenerateClassHeaderMap(const FConfigSection* UserInteropCodeGenSection, ClassHeaderMap& OutClasses)
+bool GenerateClassHeaderMap(ClassHeaderMap& OutClasses)
```

Comment:
Due to the issue fixed in UNR-273, we need to have this run on the game thread. that is why I lifted it out from the main codegen function.

Reply:
What's the reason for this to be on the game thread? I think I may have missed the context.

Reply:
It calls LoadObject to ensure that blueprints are present in memory when we run the interop codegen. LoadObject needs to be called on the game thread if end up loading assets